decidim-initiatives/spec/events/decidim/initiatives/admin/initiative_sent_to_technical_validation_event_spec.rb

```diff
@@ -3,17 +3,18 @@
 require "spec_helper"
 
 describe Decidim::Initiatives::Admin::InitiativeSentToTechnicalValidationEvent do
-  let(:resource) { create :initiative }
+  include_context "when a simple event"
+
+  let(:resource) { create :initiative, title: generate_localized_title(:initiative_title) }
```

Comment:
This is actually needed, otherwise, the pipeline will fail. 
![image](https://github.com/decidim/decidim/assets/105683/d10c611a-d920-419a-9923-96878626c6e6)